src/features/rewards/panelWelcome/style.ts

```diff
@@ -100,3 +101,24 @@ export const StyledErrorMessage = styled<{}, 'span'>('span')`
   line-height: 28px;
   color: #FFF;
 `
+
+export const StyledTOSWrapper = styled<{}, 'div'>('div')`
+  display: block;
+  margin: 20px auto;
+  width: 300px;
+`
+
+export const StyledServiceText = styled<{}, 'span'>('span')`
+  color: rgba(255, 255, 255, 0.65);
+  font-size: 14px;
+  font-family: Muli, sans-serif;
```

Comment:
please use theme font definition

Reply:
@NejcZdovc we use this exact rule (`font-family: Muli, sans-serif;`) across may different components, what is the definition you are expecting to see?